FAQ.md

```diff
@@ -87,29 +89,33 @@ This is set via the MCO’s config, not the Machine-api. The config a Machine re
 ## How does the user-data get created for the VMs?
 The user-data is generated by the installer.  The Machine-api is just a consumer of the user-data and simply passes the data to the cloud.  Refer to documentation for the Machine-config-operator (not part of the Machine-api) for creating new Machine pools and user-data.
 
-## Adding Labels to Nodes via Machines
-Labels are only added to Nodes via Machines at creation time.  You can add arbitrary labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent labels on the Machine object (or in a MachineSet’s template) will have no effect.
+## Adding Annotations and Labels to Nodes via Machines
+Annotations and Labels are only added to Nodes via Machines at creation time.  You can add arbitrary annotations and labels which are applied to Nodes immediately after they join the cluster.  Adding subsequent annotations and labels on the Machine object (or in a MachineSet’s template) will have no effect.
 
-### Which labels get added to a Node?
-When we talk about Node labels here, it can be somewhat confusing.
+### Which Annotations and Labels get added to Nodes?
+MachineSets can be used to help mark the Machines and Nodes that are created from them by using Annotations and Labels. To do this you will need to add this information in various places on your MachineSet depending on where you would like this information. Please note that this information will only be applied to new Machines and Nodes created from the MachineSet, they will not be retroactively applied.
 
-To add labels to a Node on a MachineSet, the place where you should add labels is **.spec.template.spec.metadata.labels.**
+Setting labels or annotations in `MachineSet.Spec.Template.Metadata` will cause them to be applied to every Machine object created from that MachineSet. Specifically these labels and annotations will end up in `Machine.Metadata`.
```

Comment:
Up to this point, `Labels` and `Annotations` have been capitalised, should we be consistent? 

In other places, we haven't included the type in field path strings, I think they have also been lowercased to reflect the yaml or json representation, eg `.spec.template.metadata`, should we stick with that convention or do you think this is more readable?

Reply:
good questions, i think capitalizing the proper nouns for specific resources is probably a nice readability upgrade. but i could either way.

as for the field path strings, i like including the type just to be explicit about what resource is being referenced. i think i borrowed the capitalization from code or something, i can certainly revert to lowercase if it's easier. i should probably do that just to make the document consistent. i will also look at the openshift docs to see how they approach this.